### DISCUSSION TITLE
no .gestion folder to add logo.png

### DISCUSSION THREAD

**Discussion by podrouzek** [baimard/gestion#283] [Maybe a bug ?] [answered]
I have nextcloud on ubuntu and using nginx.  
I cannot find any .gestion folder to add my logo.png file for the company logo.
Am I missing something?
![d](https://user-images.githubusercontent.com/7397587/187752242-4d8f7fe2-2122-4e48-88d5-268f099b0671.png)

**Comment by baimard** [comment, marked as answer]
Hello this is not a bug. please see that issue : #271

maybe I need to clarifie the documentation.

**Reply by podrouzek** [reply]
Ok, thank you.  Will try that and let you know

**Comment by podrouzek** [comment]
Nope, doesnt work.  I'll carry on trying.  
Nice product apart from some stuff that can be tweaked.  Work in progress I guess.
Thank you
